### PR TITLE
Clean up and grooming

### PR DESCRIPTION
**Pull request by y0sher**

### CODE REVIEW

**Comment by y0sher** [comment]
hi @pavelkrolevets can you rebase on latest `main` there were a lot of changes to the readme by @RaekwonIII

**Comment by pavelkrolevets** [comment]
@y0sher @RaekwonIII 
- added RSA + password generation. If password file is supplied then use to encrypt RSA private
- code commenting
- some readme fixes

File: cli/flags/flags.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func GetNonceFlagValue(c *cobra.Command) (uint64, error) {
 
 // NetworkFlag  adds the fork version of the network flag to the command
 func NetworkFlag(c *cobra.Command) {
-	AddPersistentStringFlag(c, network, "mainnet", "Network name: mainnet, prater, or now_test_network", false)
+	AddPersistentStringFlag(c, network, "mainnet", "Network name: mainnet, prater, or pyrmont", false)
```
Comment (y0sher) [inline cli/flags/flags.go:108]: we should probably now add holesky..

File: cli/initiator/initiator.go [hunk 2]
```diff
@@ -270,12 +285,12 @@ var StartDKG = &cobra.Command{
 		switch network {
 		case "prater":
 			forkHEX = [4]byte{0x00, 0x00, 0x10, 0x20}
+		case "pyrmont":
```
Comment (y0sher) [inline cli/initiator/initiator.go:288]: as I said, lets add holesky already..

File: cli/initiator/initiator.go [hunk 3]
```diff
@@ -270,12 +285,12 @@ var StartDKG = &cobra.Command{
...
+			forkHEX = [4]byte{0x00, 0x00, 0x20, 0x09}
 		case "mainnet":
 			forkHEX = [4]byte{0, 0, 0, 0}
-		case "now_test_network":
-			forkHEX = [4]byte{0x99, 0x99, 0x99, 0x99}
 		default:
-			logger.Fatal("😥 Please provide a valid network name: mainnet, prater, or now_test_network")
+			logger.Fatal("😥 Please provide a valid network name: mainnet/prater/pyrmont")
```
Comment (y0sher) [inline cli/initiator/initiator.go:293]: holesky

File: pkgs/dkg/drand.go [hunk 4]
```diff
@@ -487,14 +500,46 @@ func GetNetworkByFork(fork [4]byte) eth2_key_manager_core.Network {
...
+	rawKey := secretKeyBLS.SerializeToHexStr()
+	encryptedSecretShare, err := o.EncryptFunc([]byte(rawKey))
+	if err != nil {
+		return fmt.Errorf("cant encrypt private share")
+	}
+	data := shareStorage{
+		Index:  index,
+		Secret: hex.EncodeToString(encryptedSecretShare),
+	}
+	err = utils.WriteJSON(outputPath+"secret_share_"+fmt.Sprintf("%d", data.Index)+"_"+validatorPubKey.SerializeToHexStr(), &data)
```
Comment (y0sher) [inline pkgs/dkg/drand.go:540]: I can't see where this is loaded? also lets save move `secret_share` to a constant and create the whole string using `fmt.Sprintf`

File: pkgs/dkg/drand.go [hunk 5]
```diff
@@ -101,9 +105,10 @@ type LocalOwner struct {
 	RSAPub      *rsa.PublicKey
 	Owner       common.Address
 	Nonce       uint64
-	done        chan struct{}
+	Done        chan struct{}
```
Comment (y0sher) [inline pkgs/dkg/drand.go:108]: I don't get why you made all the these members public? unless they are needed for external usage lets make them private again.

**Comment by pavelkrolevets** [comment]
Merged this PR to  https://github.com/bloxapp/ssv-dkg/pull/16
Closing. Lets continue there.

**Pull request closed without merging**